src/toolwidgets.cpp

```diff
@@ -539,19 +539,24 @@ void CustomWidgetList::showWidgets()
 	foreach(QAction *act, toolbar->actions()) {
 		toolbar->removeAction(act);
 	}
-	for (int i = 0; i < widgets.size(); i++)
+	bool foundFirstNonHiddenWidget = 0;

```

Comment:
a boolean is true or false (don't use 0)

Reply:
@sunderme resolved by latest update